addons/sale_loyalty/static/src/views/fields/select_reward_radio/select_reward_radio.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module **/
```

Comment:
This file needs to be prettified
- imports should have a space around brackets `import { onWillStart } from "@odoo/owl";`
- Remove use of double new line